config/initializers/edition_services.rb

```diff
@@ -1,7 +1,11 @@
 Whitehall::Application.config.to_prepare do
   Whitehall.edition_services.tap do |coordinator|
-    coordinator.subscribe do |_event, edition, _options|
-      ServiceListeners::AttachmentUpdater.call(attachable: edition)
+    coordinator.subscribe do |event, edition, _options|
+      ServiceListeners::AttachmentUpdater.call(attachable: edition) unless %w[publish force_publish].include?(event)
+    end
+
+    coordinator.subscribe(/^(force_publish|publish)$/) do |_event, edition, _options|
+      ServiceListeners::AttachmentAssetPublisher.call(edition)
```

Comment:
Seems to me that this would be clearer as a conditional within one `coordinator.subscribe` block, otherwise it's not clear that we're going for an either/or situation.

```suggestion
    coordinator.subscribe do |event, edition, _options|
      if %w[publish force_publish].include?(event)
        ServiceListeners::AttachmentAssetPublisher.call(edition)
      else
        ServiceListeners::AttachmentUpdater.call(attachable: edition)
      end
    end
```